### ISSUE TITLE
Improve Monorepository Support

### ISSUE THREAD

**Issue by rawkode** [orhun/git-cliff#969]
### Is there an existing issue or pull request for this?

- [X] I have searched the existing issues and pull requests

### Feature description

Instead of having to execute commands, like so:

`git cliff --include-path "projects/rawkode.academy/zulip/**/*" --repository ../../../`

I'd like to instead have a `cliff.yaml` nested throughout my monorepository that allows me to segment my monorepository.

### Desired solution

Adding a `cliff.yaml` to a directory should automatically set `--include-paths` to that directory and lower.

git-cliff should always use the Git root as `--repository`

### Alternatives considered

N/A

**Comment by rawkode** [comment]
Awesome. I'll run some tests tonight. 

Thank you!

**Comment by fsimkovic** [comment]
Is testing still required on this issue? It would be good to get this feature released.

**Comment by rawkode** [comment]
Hey @orhun, it would be awesome to cut a release with this 😄

**Comment by orhun** [comment]
Yup, this should be good now. I'll try to cut a release this week 🤞🏼

**Comment by orhun** [comment]
2.8.0 is released now!
